cluster-stamp.bicep

```diff
@@ -1817,7 +1817,8 @@ resource mc 'Microsoft.ContainerService/managedClusters@2023-02-02-preview' = {
       enabled: false // Using Microsoft Entra Workload IDs for pod identities.
     }
     autoUpgradeProfile: {
-      upgradeChannel: 'stable'
+      nodeOSUpgradeChannel: 'SecurityPatch'
```

Comment:
```suggestion
      nodeOSUpgradeChannel: 'NodeImage'
```

Reply:
I could reproduce this @skabou and solved this "for now" by reverting SecurityPatch upgrades in favor of NodeImage.